src/en/TravisTranslations.lua

```diff
@@ -0,0 +1,226 @@
+-- {"id":4302,"ver":"1.0.10","libVer":"1.0.0","author":"MechTechnology"}
```

Comment:
The url dependency is missing and imo should start at version 1.0.0. Not that it really makes a difference.

Reply:
Not sure what you mean here thou about the url dependency, in all the other extensions that don't depend on libs like Madara there is no dep attribute.